### ISSUE TITLE
Issues for Real Estate (until issues section is opened)

### ISSUE THREAD

**Issue by NullCase** [NullCase/GPRealEstate#7]
@EtienneDx

Until an issues section has been created here are the issues so far.

---------------

1. Remove rental sign when rental begins. Instead, mail both players using the following format.

**Sale**

/mail send [seller] &f[&6Real Estate&f] &a[Player name] &bhas purchased your claim at &a[coordinates] &bfor &a[Sale price]&b.

**Lease**

/mail send [Landlord] &f[&6Real Estate&f] &a[Player name] &bhas accepted your lease purchase agreements at &a[coordinates] &bfor &a[#of payments] payments at &a[Lease price] &bper payment. The next payment is due in &a[Time till next payment]&b.

/mail send [Leaseholder] &f[&6Real Estate&f] &aYou have accepted &a[Landlord]&b's lease purchase agreements at &a[coordinates] &bfor &a[number of payments] payments at &a[lease price] &bper payment. The next payment is due in &a[time till next payment]&b.

**Rent**

/mail send [Landlord] &f[&6Real Estate&f] &a[Player name] &bhas accepted your rental offer at &a[coordinates] &bfor &a[number of payments] payments at &a[rental price] &bper payment. The next payment is due in &a[time till next payment]&b.

/mail send [Tennant] &f[&6Real Estate&f] &aYou have accepted &a[Landlord's player name] rental offer at &a[coordinates] &bfor &a[number of payments] payments at &a[rental price] &bper payment. The next payment is due in &a[time till next payment]&b.

---------------
2.  Sign header forces [brackets]. Permit 'Real Estate' as an option for sign header (matches Auctions).
![2019-04-29_14 08 32](https://user-images.githubusercontent.com/32785664/56917693-29c02480-6a8a-11e9-9341-0f4fdf49c5dd.png)

---------------
3. Format $ amounts as appears below.
![2019-04-29_14 25 57](https://user-images.githubusercontent.com/32785664/56918290-9be53900-6a8b-11e9-989a-699d7c6c2597.png)

--------------
4. Rental trustlist vanished on server restart. Cause unknown. (I am using my online UUID and an offline UUID).

-------------
5. Claim owner & Claim renter confused if the same name is used.

------------
6. Rental claim can be resized while rented.
Test: Create rental claim, rent it, attempt to resize.
Result: Claim resizes.
Expected: Claim cannot be resized.

**Comment by NullCase** [comment]
OK, best not to rush for this, so I will stop for today. Thanks @EtienneDx

**Comment by NullCase** [comment]
Left to test:

- Extended features list.
- Money & claim block's managed correctly.
- Resizing subclaims.
- Sale/lease/rent of subclaims.
- more stuff...

**Comment by EtienneDx** [comment]
Hey Null,
The issues section is opened, and for the different features, I wouldn’t
call that a bug more than a usage difference.
I haven’t implemented mails yet, but it’s on the list anyway.
For the signs removal, I’ll add it to the confit but I believe it does not
fit all intended usages, so I’ll default it to false.
We may discuss any other features later, but as you know, that’s a side
project for me, so I probably won’t spend hours on features I feel aren’t
necessary (I’m sure Robo will have no problem integrating them for you if
need be).
To clarify, I will do what you stated just above, and will keep fixing
bugs, but past this week I won’t integrate much new features for a while,
for school reasons.
Hope it still fits what you had in mind though

**Comment by NullCase** [comment]
Thank you for creating an issues section!

It sounds like you want to build an excellent plugin, because you mention that removing the sign does not fit all intended usages.

I too am interested to do what I can to help.

Past this week it's school. Yep.

What do you think the mail message is not offering?

----------

Here is a picture of the Rental sign. It looks like one important usage it offers is the time remaining.

![2019-04-29_14 09 55](https://user-images.githubusercontent.com/32785664/56949703-8f032c80-6b01-11e9-993b-a9a80fe7135a.png)

**Comment by EtienneDx** [comment]
For the mails, I think it's a great Idea, just hadn't had time to implement yet ^^
Will hopefully come in the next 2 days, just has to be done :)

**Comment by NullCase** [comment]
next 2 days, OK

**Comment by NullCase** [comment]
Do you need anything from me right now to help?

**Comment by EtienneDx** [comment]
Expect testing, and checking for bugs I think it's all good :)

**Comment by NullCase** [comment]
OK, It sounds like you're aiming for wednesday. I will put aside some time to test on Wednesday and post the issues to https://github.com/EtienneDx/RealEstate/issues

**Comment by EtienneDx** [comment]
Alright, I wanted to take my time but I figured it'd be better for me to do one thing after another, so I finished the features stated above (mail and signs removal in config).

That normally means no more feature and only bog fixes for a little while :)
If you could test it, I'd be glad, altough I already did test on my side and all seams correct to me.

**Comment by EtienneDx** [comment]
Just saw you made multiple points in the issue above :/
I've fixed 1 as said above, and 2 isn't a bug, it's configurable inside the config.yml
3. Will add to the list of future features
4. Yeah that's my bad, solved in 0.1.2
5. Where exactly is the message confusing? I agree it's not the best, but couldn't find better...
6. Added this to the issues section already, the problem here is that it requires to modify GriefPrevention code (most likely), so I won't search into this right now
